### ISSUE TITLE
Lock window from being resized

### ISSUE THREAD

**Issue by bottleblue** [RamonUnch/AltSnap#487]
Can we have an option to lock a window from being resized (and have it persist through reboots)?  If we try to resize it maybe you can show a padlock symbol or a red circle with a slash through it to indicate it's locked.

**Comment by RamonUnch** [comment]
What would that be for? Is there a window you want to be able to move but not resize with altsnap?
You already have the option available in the action menu to disable movement. It disables both Move+Resize with AltSnap so it is not the same exactly.

In this case a resize blacklist would be better as it would remain in your config and would persist through reboots.

**Comment by RamonUnch** [comment]
So in this case a blacklist would be useful, like it can be configured once.
Also keep in mind that if a window is not resizable by default AltSnap will try to resize it. to fix this behavior, uncheck the Resize all windows option in the general tab of the config.

There is already the `AResize` blacklist for windows to always resize, I guess I can add the `NResize` list for winows to Never resize.

**Comment by bottleblue** [comment]
Yes I saw that option. Thx for considering any features. This app is already very useful. These would make it even better imo.

**Comment by RamonUnch** [comment]
Try this build:

[AltSnap1.62test13_x64.zip](https://github.com/RamonUnch/AltSnap/files/12995742/AltSnap1.62test13_x64.zip)

You need to add in the [Blacklist] section of the ini file:
```
NResize=
; List of windows that AltSnap should Never consider as resizeable
; Only applies when ResizeAll=0
```
Like in all blacklist you can use the `exename.exe` format or the `title|class` format or even the `exename.exe:title|class` and the `*` can bu used a as a joker for `title` or class. In case of videogame I advise you simply use the exename because they usually make a single window.

**Comment by bottleblue** [comment]
It works for disabling AltSnap's resizing but not Windows native resizing. Not sure Windows resize bug will be prevented.

**Comment by RamonUnch** [comment]
You can still disable windows borders to disable native resizing. (border-less mode)

**Comment by bottleblue** [comment]
How do you do that? From the registry? I was hoping to disable all resizing from the app.

**Comment by bottleblue** [comment]
Ah ok I see. Then there's probably no need for this resize blacklist. I wanted to prevent native resizing, not AltSnap's resizing.

**Comment by LoganDark** [comment]
when you make a window borderless, AltSnap can still move and resize it, but you can't anymore. I use this infrequently, but it's still a very useful feature

**Comment by RamonUnch** [comment]
if the issue is about disabling system resizing then I close as unplanned
